userspace/libpman/src/maps.c

```diff
@@ -295,6 +378,38 @@ int pman_fill_syscall_exit_extra_tail_table() {
 
 /*=============================== BPF_MAP_TYPE_ARRAY ===============================*/
 
+int pman_fill_interesting_syscalls_table_64bit() {
+	char error_message[MAX_ERROR_MESSAGE_LEN];
+	int fd = bpf_map__fd(g_state.skel->maps.interesting_syscalls_table_64bit);
+	for(uint32_t i = 0; i < SYSCALL_TABLE_SIZE; i++) {
+		const bool interesting = false;
+		if(bpf_map_update_elem(fd, &i, &interesting, BPF_ANY) < 0) {
+			snprintf(error_message,
+			         MAX_ERROR_MESSAGE_LEN,
+			         "unable to initialize interesting syscall table at index %d!",
+			         i);
+			pman_print_error((const char*)error_message);
+			return errno;
+		}
+	}
+	return 0;
+}
+
+int pman_mark_single_64bit_syscall(int syscall_id, bool interesting) {
+	char error_message[MAX_ERROR_MESSAGE_LEN];
+	int fd = bpf_map__fd(g_state.skel->maps.interesting_syscalls_table_64bit);
+	if(bpf_map_update_elem(fd, &syscall_id, &interesting, BPF_ANY) < 0) {
```

Comment:
If this is called frequently it could have an impact as well but AFAIK we don't call it so frequently at runtime